src/System.Net.NameResolution/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -21,5 +30,103 @@ public static void EventSource_ExistsWithCorrectId()
 
             Assert.NotEmpty(EventSource.GenerateManifest(esType, "assemblyPathToIncludeInManifest"));
         }
+
+        [ConditionalFact]
+        public void GetHostEntry_InvalidHost_LogsError()
+        {
```

Comment:
These should be RemoteInvoke'd.  Otherwise other tests in this process could interfere with these tests, and vice versa.

Reply:
They should not interfere. I put [Collection("NoParallelTests")] on the class so we don't need RemoteInvoke.